### DISCUSSION TITLE
Automate SO with Multiple Bonded NICS

### DISCUSSION THREAD

**Discussion by gambl3** [Security-Onion-Solutions/securityonion#5448] [Unsupported Versions] [answered]
I'm attempting to automate the installation of SO 2.3.70 using a network install and multiple bonded NICs for collection. 

I'm not sure what the correct format/delimiter is for the BNICS variable. The automation script variables are not documented anywhere that I have found.  I have tried space and comma delimited and thus far neither results in my collection NICs being added to bond0 interface. 

What is the correct way to format multiple BNICS for an automated install?

TESTING=true
address_type=DHCP
ADMINUSER=onionuser
ADMINPASS1=xxxxxxxx (redacted)
ADMINPASS2=xxxxxxxx (redacted)
ALLOW_CIDR=192.168.1.0/24
ALLOW_ROLE=a
BASICZEEK=2
BASICSURI=2
**BNICS=eno2,eno3,eno4,eno5,eno6,eno7,eno8**
ZEEKVERSION=ZEEK
GRAFANA=1
EVALADVANCED=BASIC
HNMANAGER=10.0.0.0/8,192.168.0.0/16,172.16.0.0/12
HNSENSOR=inherit
HOSTNAME=sensor1
MANAGERADV=BASIC
MNIC=eno1
NIDS=Suricata
NODESETUP=NODEBASIC
NSMSETUP=BASIC
NODEUPDATES=MANAGER
OSQUERY=1
PATCHSCHEDULENAME=auto
PLAYBOOK=1
REDIRECTINFO=IP
RULESETUP=ETOPEN
SKIP_REBOOT=1
SOREMOTEPASS1=xxxxxxxx (redacted)
SOREMOTEPASS2=xxxxxxxx (redacted)
install_type=STANDALONE
STRELKA=1
THEHIVE=1
WAZUH=1
WEBPASSWD1=xxxxxxxx (redacted)
WEBPASSWD2=xxxxxxxx (redacted)

**Comment by gambl3** [comment, marked as answer]
I solved the issue after looking into how the so-whiptail script handles the interactive install. The BNICS variable expects a bash array so using my answers file above it looks like: **BNICS=(eno2 eno3 eno4 eno5 eno6 eno7 eno8)**

**Comment by johnykex** [comment]
I'm trying to do the same thing but I'm stack. Can you give me more details how you manage to achieve it?
